Java/libraries/recognizers-text-sequence/src/main/java/com/microsoft/recognizers/text/sequence/resources/ChineseIp.java

```diff
@@ -14,17 +14,17 @@
 public class ChineseIp {
 
     public static final String Ipv4Regex = "({PhoneNumbersDefinitions.WordBoundariesRegex}(1\\d{2}|2[0-4]\\d|25[0-5]|0?[1-9]\\d|0{0,2}\\d)((\\.(1\\d{2}|2[0-4]\\d|25[0-5]|0?[1-9]\\d|0{0,2}\\d)){3}){PhoneNumbersDefinitions.EndWordBoundariesRegex})"
-            .replace("{PhoneNumbersDefinitions.WordBoundariesRegex}", PhoneNumbersDefinitions.WordBoundariesRegex)
-            .replace("{PhoneNumbersDefinitions.EndWordBoundariesRegex}", PhoneNumbersDefinitions.EndWordBoundariesRegex);
+            .replace("{PhoneNumbersDefinitions.WordBoundariesRegex}", BasePhoneNumbers.WordBoundariesRegex)
```

Comment:
Are these ok? I thought these were temporary solutions

Reply:
Here we are fixing the generated resources.. IMO the problem is the resource-generator, but with these changes we are unblocking all.

Without these changes, the solution won't compile